ucm2/NXP/iMX8/Librem_5/HiFi.conf

```diff
@@ -0,0 +1,180 @@
+		cset "name='SPKOUTR PGA' DAC"
+		cset "name='Speaker Volume' 121,121"
+		cset "name='Speaker Boost Volume' 2"
+	]
+
+	DisableSequence [
+		cset "name='Speaker Switch' off,off"
+		cset "name='DAC L/R Swap Switch' off"
+		cset "name='DAC Monomix Switch' off"
+		cset "name='SPKOUTR PGA' Mixer"
+	]
+
+	Value {
+		PlaybackChannels "2"
+		PlaybackPriority "100"
+		PlaybackVolume "name='Speaker Volume'"
+		PlaybackSwitch "name='Speaker Switch'"
+		PlaybackPCM "hw:${CardId},0"
+	}
+}
+
+SectionDevice."Speaker" {
+	Comment "Speaker"
+
+	ConflictingDevice [
+		"Handset"
+		"Headphones"
+	]
+
+	EnableSequence [
+		cset "name='Speaker Switch' on,off"
+		cset "name='DAC Monomix Switch' on"
+		cset "name='SPKOUTL PGA' DAC"
+		cset "name='Speaker Volume' 121,121"
+		cset "name='Speaker Boost Volume' 6"
+	]
+
+	DisableSequence [
+		cset "name='Speaker Switch' off,off"
+		cset "name='DAC Monomix Switch' off"
+		cset "name='SPKOUTL PGA' Mixer"
+	]
+
+	Value {
+		PlaybackChannels "2"
+		PlaybackPriority "500"
+		PlaybackVolume "name='Speaker Volume'"
+		PlaybackSwitch "name='Speaker Switch'"
+		PlaybackPCM "hw:${CardId},0"
+	}
+}
+
+SectionDevice."Headphones" {
+	Comment "Headphones"
+
+	ConflictingDevice [
+		"Handset"
+		"Speaker"
+	]
+
+	EnableSequence [
+		cset "name='Headphone Switch' on,on"
+		cset "name='HPOUTL PGA' 0 unmute"
+		cset "name='HPOUTR PGA' 0 unmute"
+		cset "name='Headphone Aux Volume' 0"
+		cset "name='Headphone Volume' 121,121"
+	]
+
+	DisableSequence [
+		cset "name='Headphone Switch' off off"
+		cset "name='HPOUTL PGA' 0 mute"
+		cset "name='HPOUTR PGA' 0 mute"
+	]
+
+	Value {
+		PlaybackPriority "1000"
+		PlaybackChannels "2"
+		PlaybackVolume "name='Headphone Volume'"
+		PlaybackSwitch "name='Headphone Switch'"
+		PlaybackPCM "hw:${CardId},0"
+		JackControl "Headphones Jack"
+	}
+}
+
+SectionDevice."Mic" {
+	Comment "Internal Microphone"
+
+	ConflictingDevice [
+		"Headset"
+	]
+
+	EnableSequence [
+		cset "name='Digital Capture Volume' 116,116"
+		cset "name='ADC L/R Swap Switch' on"
+		cset "name='Input Mode' Digital"
+	]
+
+	DisableSequence [
+		cset "name='Digital Capture Volume' 0,0"
+		cset "name='ADC L/R Swap Switch' off"
+	]
+
+	Value {
+		CapturePriority "500"
+		CaptureChannels "2"
+		CaptureVolume "name='Digital Capture Volume'"
+		CapturePCM "hw:${CardId},0"
+	}
+}
+
+SectionDevice."Headset" {
+	Comment "Headset Microphone"
+
+	ConflictingDevice [
+		"Mic"
+	]
+
+	EnableSequence [
+		cset "name='Digital Capture Volume' 116,116"
+		cset "name='Input Mode' Analog"
+		cset "name='MIXINR PGA Volume' 1,1"
+		cset "name='MIXINR PGA Switch' on,on"
+		cset "name='Capture Volume' 39,39"
+		cset "name='Capture Switch' on,on"
+	]
+
+	DisableSequence [
+		cset "name='Capture Switch' off,off"
+		cset "name='MIXINR PGA Switch' off,off"
+		cset "name='Input Mode' Digital"
+	]
+
+	Value {
+		CapturePriority "100"
+		CaptureChannels "2"
+		CaptureVolume "name='Capture Volume'"
+		CapturePCM "hw:${CardId},0"
```

Comment:
First (zero) PCM device is the default. Use the shorter `CapturePCM "hw:${CardId}"`